### DISCUSSION TITLE
Support for RPI 5

### DISCUSSION THREAD

**Discussion by rutra8002** [home-assistant/operating-system#3138] [General]
Hey, i was thinking about buying Raspberry PI 5, and i want to use home assistant with it. But as there is currently no stable release for RPI5, should i buy RPI 4 instead, or buy RPI5?

**Comment by PeterDeelen1** [comment]
Hello, 
I've recently bought a Raspberry PI 5, attached an SSD drive with usb c converter. When the raspberry is powered-off then you need to do a re-install. 
I now make regular backups to do a restore if needed. it only takes 5 minutes to get back-up and running with home-assistant. 
the raspberry PI is always on in my case.

So in my opinion buy the Pi5 it will be more stable soon I hope.
